data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -722,6 +722,22 @@
     ],
     "flags": [ "ALLOW_ROTTEN" ]
   },
+  {
+  "type": "recipe",
+  "result": "soap_flakes",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_CHEMICALS",
+  "skill_used": "cooking",
+  "difficulty": 1,
+  "time": "45 m",
```

Comment:
45 minutes to cut flakes?

Reply:
well you have to cut up a bar of soap into small flakes. I don't think 45 minutes is to long for that.

Reply:
Suggestion: two recipes.
* one like this (I also agree that it should be less than 15 minutes)
* second with a more suitable tool if we have it (several minutes to craft)